sig-architecture/charter.md

```diff
@@ -1,40 +1,175 @@
+
+SIG Architecture is also a good place to identify gaps in the project, collect thinking on how the project should approach those problems and then work to motivate that those problems get addressed. This will often take the form of working to draw attention through an appropriate SIG.
+
+Issues specific to a particular component or functional area, which would be the purview of some other single SIG, are out of scope for SIG Architecture, except where they deviate from project-wide principles and conventions, or are not yet being addressed, as discussed above.
+
+SIG Architecture’s subprojects are as follows. [sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml) contains links to their OWNERS.
+
+<table>
+  <tr>
+    <td>Subproject</td>
+    <td>Description</td>
+    <td>Example Artifacts</td>
+  </tr>
+  <tr>
+    <td>Kubernetes scope</td>
+    <td>Defining "what is Kubernetes", approving which repos belong in the kubernetes GitHub org, evaluating, guiding, and curating new components to be added to Kubernetes releases where the implementation is consistent with the defined architectural standards</td>
+    <td>What is (and is not) Kubernetes</td>
+  </tr>
+  <tr>
+    <td>Conformance definition</td>
+    <td>Reviewing, approving, and driving changes to the conformance test suite; reviewing, guiding, and creating new conformance profiles</td>
+    <td>Test list, Test guidelines</td>
+  </tr>
+  <tr>
+    <td>Architectural governance</td>
+    <td>Establishing and documenting design principles, documenting and evolving the system architecture, reviewing, curating, and documenting new extension patterns</td>
+    <td>Kubernetes Design and Architecture, Design principles</td>
+  </tr>
+  <tr>
+    <td>API governance</td>
+    <td>Establishing and documenting conventions for system and user-facing APIs, define and operate the APl review process, final API implementation consistency validation, co-own top-level API directories with API machinery</td>
+    <td>API conventions, API linter</td>
+  </tr>
+  <tr>
+    <td>Deprecation policy</td>
+    <td>Maintaining, evolving, and enforcing the deprecation policy</td>
+    <td>Deprecation policy</td>
+  </tr>
+  <tr>
+    <td>Code organization</td>
+    <td>Overall code organization, including github repositories and branching methodology, top-level and pkg OWNERS of kubernetes/kubernetes, kubernetes-template-project</td>
+    <td>kubernetes-template-project</td>
+  </tr>
+  <tr>
+    <td>KEP</td>
+    <td>Develop and drive technical enhancement review process</td>
+    <td>Process, Template</td>
+  </tr>
+</table>
+
+
+## Roles
+
+The following roles are required for the SIG to properly function. In the event that any role is unfilled, the SIG will make a best effort to fill it, and any decisions reliant on a missing role will be postponed until the role is filled.
+
+* **Chair**
+
+    * Number: 2-3
```

Comment:
I think we should just say "3" that way we don't have ties.

Reply:
+1